cinema/views.py

```diff
@@ -43,6 +46,28 @@ def get_serializer_class(self):
 
         return MovieSerializer
 
+    def get_queryset(self):
+        queryset = self.queryset
```

Comment:
call super method instead of accesing this attribute directly
solve N+1 problem